projects/pgai/pgai/vectorizer/embedders/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .ollama import Ollama
+from .openai import OpenAI
+from .voyageai import VoyageAI
+
+__all__ = ["Ollama", "OpenAI", "VoyageAI"]
```

Comment:
Do we need this `__all__`? According to [this](https://docs.python.org/3/tutorial/modules.html#importing-from-a-package):
>  if a package’s `__init__.py` code defines a list named `__all__`, it is taken to be the list of module names that should be imported when `from package import *` is encountered

This seems like somewhat esoteric functionality, which we're not actively using, and I don't see the need to support. By removing it we can remove one more instruction in the readme above.

Reply:
I made this in order to be able to import all providers through the same module: `from .embedders import Ollama, OpenAI, VoyageAI`  The alternative (unless you know another one) would be to use individual imports such as:

```python
from .embedders.ollama import Ollama
from .embedders.openai import OpenAI
from .embedders.voyageai import VoyageAI
```

Reply:
Thanks for the explanation! I don't have a strong opinion, but the second suggestions (imports in `__init__.py`) is "less magical", so I would prefer it over `__all__`.